apps/ledger-live-mobile/jest.config.js

```diff
@@ -36,7 +36,12 @@ module.exports = {
   ],
   testPathIgnorePatterns: ["<rootDir>/node_modules/"],
   moduleDirectories: ["node_modules"],
-  coverageReporters: ["json"],
+  collectCoverage: false,
```

Comment:
I had to turn off collecting coverage : it is breaking on the CI. Will investigate the issue in another ticket.